### ISSUE TITLE
Player does not stop on mobile

### ISSUE THREAD

**Issue by ahmadabdolsaheb** [freeCodeCamp/coderadio-client#126]
Although the tapping the play button starts the music, taping it again does not stop the music.

ios: 14.3
browser: 87.0.4280.77

**Comment by ojeytonwilliams** [comment]
It does behave as expected on 
Android: 7.0
Chromium: 88.0.4324.96

**Comment by huyenltnguyen** [comment]
This might sound strange, but do you think it would work if we change our 'div' button to a real `<button>`?

https://github.com/freeCodeCamp/coderadio-client/blob/ecd8e83466b9d22d460162bd4225a35a23c62cca/src/components/PlayPauseButton.js#L35-L46

I don't understand HTML very well, but I believe the `button` element itself has some default behaviors built-in, which might be missing from the `div` button. I don't have an iOS device to test, so I guess I'm just throwing things at the wall here.

But why do weird things just keep happening to us? :cry:

**Comment by ahmadabdolsaheb** [comment]
Yes, that makes sense @huyenltnguyen, and it would be more accessible.
